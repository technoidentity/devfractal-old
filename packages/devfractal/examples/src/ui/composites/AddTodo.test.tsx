import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { SimpleAddTodo } from './AddTodo'

test('AddTodo', () => {
  const component: ReactTestRenderer = renderer.create(<SimpleAddTodo />)
  expect(component.toJSON()).toMatchSnapshot()
})
