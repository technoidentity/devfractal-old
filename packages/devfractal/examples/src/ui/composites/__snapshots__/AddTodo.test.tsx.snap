// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTodo 1`] = `
<section
  className="section"
>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label
        className="label"
        htmlFor="title"
      >
        Title:
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          id="title"
          name="title"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="label"
        htmlFor="description"
      >
        Description:
      </label>
      <div
        className="control"
      >
        <textarea
          className="textarea"
          id="description"
          name="description"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="label"
        htmlFor="done"
      >
         Done
      </label>
      <div
        className="control"
      >
        <label
          className="checkbox"
        >
          <input
            checked={false}
            className="checkbox"
            id="done"
            name="done"
            onBlur={[Function]}
            onChange={[Function]}
            type="checkbox"
            value={false}
          />
        </label>
      </div>
    </div>
    <div
      className="buttons"
    >
      <button
        className="button is-info"
        disabled={false}
        type="submit"
      >
        Submit
      </button>
      <button
        className="button is-danger"
        disabled={true}
        onClick={[Function]}
        type="reset"
      >
        Reset
      </button>
    </div>
    <pre>
      {
  "title": "",
  "description": "",
  "done": false
}
    </pre>
  </form>
</section>
`;
